packages/json-rpc/json-rpc.0.5.0-0/opam

```diff
@@ -0,0 +1,22 @@
+opam-version: "1.2"
+maintainer: "Markus W. Weissmann <markus.weissmann@in.tum.de>"
+authors: [ "Markus W. Weissmann <markus.weissmann@in.tum.de>" ]
+license: "MIT"
+homepage: "https://github.com/mwweissmann/ocaml-json-rpc"
+dev-repo: "git://github.com/mwweissmann/ocaml-json-rpc.git"
+bug-reports: "https://github.com/mwweissmann/ocaml-json-rpc/issues"
+build: [
+  ["ocaml" "setup.ml" "-configure" "--prefix" prefix]
+  ["ocaml" "setup.ml" "-build"]
+]
+install: ["ocaml" "setup.ml" "-install"]
+remove: [
+  ["ocamlfind" "remove" "json-rpc"]
+]
+build-doc: [ "ocaml" "setup.ml" "-doc" ]
+depends: [
+  "result"
+  "yojson"
+  "ocamlfind" {>= "1.5"}
+  "ocamlbuild" {build}
+]
```

Comment:
This appears to require an ocaml compiler version constraint of >4.02.3

```
available: [ ocaml-version >= "4.03.0" ]
```
will do the trick.